.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,297 @@
+
+          $bootstrapOptions = "-EnableVerbose -EnableStaticTileDB -EnableBuildDeps"
+          if ($env:TILEDB_S3 -eq "ON") {
+            $bootstrapOptions = "-EnableS3 " + $bootstrapOptions
+          }
+          if ($env:TILEDB_AZURE -eq "ON") {
+            $bootstrapOptions = "-EnableAzure " + $bootstrapOptions
+          }
+          if ($env:TILEDB_GCS -eq "ON") {
+            $bootstrapOptions = "-EnableGCS " + $bootstrapOptions
+            #NOTE: GCS simulator not yet actually in place.
+          }
+          if ($env:TILEDB_SERIALIZATION -eq "ON") {
+            $bootstrapOptions = "-EnableSerialization " + $bootstrapOptions
+          }
+          # static already added above as initial default
+          # if ($env:TILEDB_STATIC -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableStaticTileDB"
+          # }
+          # if ($env:TILEDB_HDFS -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableHDFS"
+          #}
+          if ($env:TILEDB_TOOLS -eq "ON") {
+            $bootstrapOptions = $bootstrapOptions + " -EnableTools"
+          }
+          if ($env:TILEDB_DEBUG -eq "ON") {
+            $bootstrapOptions = $bootstrapOptions + " -EnableDebug"
+          }
+          # if ($env:TILEDB_CI_ASAN -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableSanitizer address -EnableDebug"
+          # }
+          # if ($env:TILEDB_CI_TSAN -eq "ON") {
+          #  $bootstrapOptions = $bootstrapOptions + " -EnableSanitizer thread -EnableDebug"
+          # }
+          # static already added above as initial default
+          # if ($env:TILEDB_FORCE_BUILD_DEPS" -eq "ON") {
+          #   $bootstrapOptions = $bootstrapOptions + " -EnableBuildDeps"
+          # }
+          if ($env:TILEDB_WERROR -eq "OFF") {
+            $bootstrapOptions = $bootstrapOptions + " -DisableWerror"
+          }
+          
+          $bootstrapExpression = "& $env:BUILD_SOURCESDIRECTORY\bootstrap.ps1 " + $bootstrapOptions
+          Write-Host "bootstrapExpression: $bootstrapExpression"
+          Invoke-Expression $bootstrapExpression
+
+          if ($LastExitCode -ne 0) {
+            Write-Host "Bootstrap failed."
+            exit $LastExitCode
+          }
+
+          cmake --build $env:BUILD_BUILDDIRECTORY --config Release -j $env:NUMBER_OF_PROCESSORS
+
+          if ($LastExitCode -ne 0) {
+            Write-Host "Build failed. CMake exit status: " $LastExitCocde
+            exit $LastExitCode
+          }
+
+          cmake --build $env:BUILD_BUILDDIRECTORY  -j $env:NUMBER_OF_PROCESSORS --target install-tiledb --config Release 2>&1
+
+          if ($LastExitCode -ne 0) {
+            Write-Host "Installation failed."
+            $host.SetShouldExit($LastExitCode)
+          }
+        shell: powershell
+
+      - name: 'Test'
+        shell: powershell
+        run: |
+          write-host "begin run: 'Test'"
+
+          $env:MINIO_ACCESS_KEY = "minio"

```

Comment:
Sorry if naive comment. In `Linux/Macos` implementations these `envs` are produced by `run-minio`scripts. Is it an overlap here? Are we planning to have them inside the jobs from now on or are these scripts still in use? If so wouldn't it make sense to extend the script for installing and running `minio` in WIN as well?

Reply:
I do not know the history of these, but a brief exploration finds that
A) there is no run-minio.ps1 (windows version)
B) the vars are (also) set in the install-minio.ps1 script
C) tests using minio aren't actually being run for windows ATM

Reply:
Hmm then I guess it will be a design choice on either deciding to keep these scripts across platforms or refactoring at some point so all of them follow the same design principle on installing runner's dependencies and exporting env vars. In any case I just pinpoint it consider me covered review-wise :)

Reply:
IMO we should keep this stuff in one place, as above preferably in a script wherever possible rather than in-line YAML. I wrote `install-minio.ps1` and it didn't seem necessary to have a separate `run-` script for the use-case, but no objection to separating the run function if desired.